### PR TITLE
arch/stm32l4: Fix nxstyle errors

### PR DESCRIPTION
**Pull request by hartmannathan**
## Summary

Fix (most) nxstyle errors in:
- arch/arm/include/stm32l4/stm32l4x3xx_irq.h
- arch/arm/include/stm32l4/stm32l4xrxx_irq.h

## Impact

Removes nxstyle errors.

## Testing

nxstyle

### CODE REVIEW

**Comment by hartmannathan** [comment]
Any thoughts on how to handle the /* Reserved */ comments that are throwing the nxstyle "Wrong column position or missing blank line before comment" errors?

**Comment by jerpelea** [comment]
@hartmannathan usually blank line before that line

File: arch/arm/include/stm32l4/stm32l4x3xx_irq.h [hunk 1]
```diff
@@ -113,7 +117,7 @@
 /* Reserved */                                           /* 50: TIM5 global interrupt */
 #define STM32L4_IRQ_SPI3        (STM32L4_IRQ_FIRST + 51) /* 51: SPI3 global interrupt */
 #define STM32L4_IRQ_UART4       (STM32L4_IRQ_FIRST + 52) /* 52: UART4 global interrupt */
-/* Reserved */                                         /* 53: UART5 global interrupt */
+/* Reserved */                                           /* 53: UART5 global interrupt */
```
Comment (davids5) [inline arch/arm/include/stm32l4/stm32l4x3xx_irq.h:120]: --------------------------------ws----------------------------------- /* Reserved 53: UART5 global interrupt */

Comment (jerpelea) [reply arch/arm/include/stm32l4/stm32l4x3xx_irq.h:120]: --------------------------------ws----------------------------------- /* 53: UART5 global interrupt - Reserved */

Comment (hartmannathan) [reply arch/arm/include/stm32l4/stm32l4x3xx_irq.h]: Thanks! Done

**Review by xiaoxiang781216 — APPROVED** [review]
LGTM.

**Review by davids5 — APPROVED** [review]
@jerpelea - It is Ok as done?

**Comment by jerpelea** [comment]
LGTM

**Pull request merged**